### DISCUSSION TITLE
入库工具 提示 入库成功后，为什么重启Steam后，Steam 中不显示 入库的游戏？

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#7] [Question / Answer] [answered]
使用 SAI 或 其他入库工具后，显示 入库成功。
但是 为什么重启Steam后，Steam 中不显示 入库的游戏？

**Comment by pjy612** [comment]
目前 新版 Steam 默认不显示 假入库的游戏。  
但是 显示 已安装 的 游戏。  
所以入库完毕后，在 配套解锁工具 功能正常的情况下。   
使用以下方式来触发安装操作：  
+ 可以通过 `开始 -> 运行` 输入 `steam://install/appid` 来触发安装操作。
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/4875080e-cf70-4d86-a389-40eda7a7f321)

+ 可以通过 SteamDb 指定 游戏页面 上 的 `install` 按钮，触发安装。
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/72cc44ae-05b2-495c-ae01-c2a82d5b4b84)

PS. 
如果 入库工具提示成功，但 是通过 上述步骤 在安装时 遇到 `无许可` 的情况  
大概率是 配套的解锁 工具功能失效 或 使用姿势不正确

**Reply by pjy612** [reply]
反正 这也算是一种好调整。
毕竟 总有人先问怎么 入库，之后又问为什么移除不了。。。
这种调整反而最好，安装了就有，卸载直接消失。

**Comment by mzt0107-arch** [comment]
显示无许可怎么办

**Reply by pjy612** [reply]
问解锁工具去

**Comment by wangzhining1235** [comment]
解锁工具是什么老大

**Reply by pjy612** [reply, marked as answer]
这多久的坟了？解锁工具 SteamTools，OpenSteamTools。
SAI 目前已 自带 SAI Core 解锁内核。
